### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by taraojo**
Fixes some out of date information and messaging that is no longer relevant or working.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,15 +75,24 @@ The above `nginx` configuration proxies `https://localhost:4437` to `http://loca
...
 
 ### Shared storage
 
-To run this demo, [follow the instructions](https://developer.chrome.com/docs/privacy-sandbox/shared-storage/#try-the-shared-storage-api) for enabling the Privacy Sandbox Ads APIs experiment flag at **chrome://flags/#privacy-sandbox-ads-apis**.
+To run this demo, enable the experiment flags:
+
+* chrome://flags/#enable-fenced-frames
+* chrome://flags/#privacy-sandbox-enrollment-overrides
+
+Then, open Google Chrome from the command line with the additional arguments as below to disable the need for attestations when running locally.
```
Comment (kevinkiklee) [inline README.md:89]: Then, open [Google Chrome from the command line](https://www.chromium.org/developers/how-tos/run-chromium-with-flags/) with the...

File: README.md [hunk 2]
```diff
@@ -75,15 +75,24 @@ The above `nginx` configuration proxies `https://localhost:4437` to `http://loca
...
+
+```bash
+open /Applications/Google\ Chrome\ Canary.app --args --enable-privacy-sandbox-ads-apis --disable-features=EnforcePrivacySandboxAttestations
```
Comment (kevinkiklee) [inline README.md:92]: 1) This command (`open`) only works on a Mac. 
2) We should recommend the Stable release over Canary

Let's just simplify this line to `--enable-privacy-sandbox-ads-apis --disable-features=EnforcePrivacySandboxAttestations`.

File: README.md [hunk 3]
```diff
@@ -75,15 +75,24 @@ The above `nginx` configuration proxies `https://localhost:4437` to `http://loca
...
+```bash
 npm run dev
 ```
 
 ### Shared storage
 
-To run this demo, [follow the instructions](https://developer.chrome.com/docs/privacy-sandbox/shared-storage/#try-the-shared-storage-api) for enabling the Privacy Sandbox Ads APIs experiment flag at **chrome://flags/#privacy-sandbox-ads-apis**.
+To run this demo, enable the experiment flags:
+
+* chrome://flags/#enable-fenced-frames
```
Comment (kevinkiklee) [inline README.md:86]: Hmm. I have the FF flag set as "Default" and it works properly. Would you double check that you need the FF flags to be changed for the demo to run?

Comment (taraojo) [reply README.md]: Oh cool, it does work fine without this flag. Is it possible the other flag also enables fenced frames?

File: README.md (hunk 2 above)
Comment (taraojo) [reply README.md]: Got it, thanks!

File: README.md (hunk 3 above)
Comment (kevinkiklee) [reply README.md]: Yea, that's possible but I'm not 100% sure.

**Review by kevinkiklee — APPROVED** [review]

**Pull request merged**